assets/css/base/_base.scss

```diff
@@ -1 +1,52 @@
 // Base styles.
+
+// Typography
+h1 {
+	font-size: 4em ;
+	font-weight: semi-bold ;
+}
+h2 {
+	font-size: 3em ;
+	font-weight: medium ;
+}
+h3 {
+	font-size: 1.5em ;
+	font-weight: bold ;
+}
+p {
+	font-size: 1em ;
+	font-weight: regular ;
+}
+
+// Links and Buttons
+a {
+	color: #6895c6;
+	font-weight: bold;
+	text-decoration: none;
+}
+a:hover,
+a:focus {
+	background-color: #FADC90;
+}
+
+button {
+	background-color: #cee6c5;
+	border: 1px solid #8CAABB;
+	border-radius: 70px;
+	box-sizing: border-box;
```

Comment:
You should add this as a global rule. See: https://css-tricks.com/box-sizing/

Reply:
Like this?

```css
* {
  box-sizing: border-box;
}
```

Reply:
Yep!